src/Protocol/DFRN.php

```diff
@@ -2141,7 +2141,7 @@ private static function updateContent($current, $item, $importer, $entrytype)
 
 			$changed = true;
 
-			if ($entrytype == DFRN_REPLY_RC) {
+			if ($entrytype == DFRN::DFRN_REPLY_RC) {
```

Comment:
It should be `DFRN::REPLY_RC` instead, can you please rename all `DFRN_*` constants to just `*` and modify their use?

Reply:
Done https://github.com/friendica/friendica/pull/5036/commits/736c76d068742a4e93bef77d1f23a0c9e23417be